optaplanner-website-root/content/competitor/gurobi.adoc

```diff
@@ -51,8 +51,7 @@ for customers who want to take their OptaPlanner implementation to the next leve
 
 // TODO Change to "OptaPlanner is both faster and more scalable than Gurobi" after adding the open benchmarks as proof.
 OptaPlanner is both fast and scalable
-for planning and scheduling use cases, such as the link:../learn/useCases/vehicleRoutingProblem.html[Vehicle Routing Problem],
-link:../learn/useCases/employeeRostering.html[Employee Rostering] and link:../learn/useCases/maintenanceScheduling.html[Maintenance scheduling].
+for planning and scheduling use cases, such as the link:../learn/useCases/vehicleRoutingProblem.html[Vehicle Routing Problem] and link:../learn/useCases/maintenanceScheduling.html[Maintenance scheduling].
```

Comment:
"link:../learn/useCases/employeeRostering.html" is not optaweb-employee-rostering

It's the landing page for "employee rostering" to explain the use case, host the employee rsotering video (which will soon use our optaplanner-quickstarts variant) and to link to the optaplanner-qcuikstarts employee scheduling impl (instead of optaweb)

That landing page is very import SEO wise.